### PR TITLE
Add redis plugin

### PR DESCRIPTION
**Pull request by kennedyoliveira**
Thank you for your submission to the FreeNAS community plugins repository!

Please ensure the following guidelines are met when submitting a new Plugin.

1. Add the entry to INDEX in alphabetical order
2. Includes a new icon in the ./icon/ directory
3. Ensure the submitted icon file is a valid PNG that is 128x128 in size
4. Add new plugin cirrus task

### CODE REVIEW

**Comment by fulder** [comment]
LGTM and both checks are green :+1: just a minor (and stupid) question: Whats the reason for installing redis from source rather than using the already prebuilt `redis-6.0.14` port (except it being a bit older than the source version)?

**Comment by kennedyoliveira** [comment]
The question is not stupid at all, but you already answered yet , it's for easy of update and keeping on the latest version.

File: redis.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Redis",
+    "release": "12.2-RELEASE",
+    "artifact": "https://github.com/kennedyoliveira/iocage-redis-plugin",
+    "properties": {
+        "dhcp": 1,
+        "boot": "on"
```
Comment (sonicaj) [inline redis.json:7]: This is not really required and is on by default

**Review by sonicaj — CHANGES_REQUESTED** [review]
@kennedyoliveira you forgot to add redis to INDEX ;)

File: redis.json (hunk 1 above)
Comment (kennedyoliveira) [reply redis.json]: ah ok, removed.

**Comment by kennedyoliveira** [comment]
> @kennedyoliveira you forgot to add redis to INDEX ;)

True lol, should not do this stuff when you are almost falling asleep, added, thanks!

**Review by sonicaj — APPROVED** [review]
Thank you for your contribution @kennedyoliveira!

**Pull request merged**